src/commands/yoink.ts

```diff
@@ -85,7 +90,15 @@ export default class extends CommandModel<PreHandlers> {
 
 		const promises: Promise<[string, ChangeEmoteInset]>[] = [];
 
-		toAdd.forEach((emote) => promises.push(addEmote(emote, EmoteSet(), keepAlias)));
+		const emoteset: string | undefined = (await gql.GetUser(await Bot.User.ResolveUsername(srcUser.username))).connections.find(i => i.platform === ConnectionPlatform.TWITCH)?.emote_set_id
```

Comment:
unnecessary call to 7tv. We already have the user fetched `:58`. And im pretty sure the code above prevents `:95` from being required.